src/config.rs

```diff
@@ -179,6 +183,14 @@ pub const ENV_OUTBOUND_ROUTER_CAPACITY: &str = "LINKERD2_PROXY_OUTBOUND_ROUTER_C
 pub const ENV_INBOUND_ROUTER_MAX_IDLE_AGE: &str = "LINKERD2_PROXY_INBOUND_ROUTER_MAX_IDLE_AGE";
 pub const ENV_OUTBOUND_ROUTER_MAX_IDLE_AGE: &str = "LINKERD2_PROXY_OUTBOUND_ROUTER_MAX_IDLE_AGE";
 
+
+/// Limits the maximum number of outbound Destination service queries.
+///
+/// Routes which do not result in service discovery lookups will not be capped
+/// by this limit. This will have no effect if it is greater than the total
+/// router capacity (as configured by `ENV_OUTBOUND_ROUTER_CAPACITY`).
+pub const ENV_MAX_DESTINATION_QUERIES: &str = "LINKERD2_PROXY_MAX_DESTINATION_QUERIES";
```

Comment:
Similarly, _DESTINATION_CONCURRENCY_LIMIT

Reply:
Yeah, that sounds good to me. I'm not terribly attached to the current name; picked it mainly because i wanted _something_ better than the _original_ working name, `OUTBOUND_MAX_RESOLUTIONS`, which was obviously wrong